BLSQ/iaso
# POLIO-1702:  pdf preview

We need to display a preview of pdf on:
- Stock variations (Form A, Destruction Report, Incident Report)
- Supply chain  (VRF, PRE-ALERT)

Related JIRA tickets :  POLIO-1702
## Self proofreading checklist


## Changes

- installed react-pdf
- create a dedicated component to preview a pdf
- create a dedicated component to pdf file upload
- replace it a a bit everywhere

## How to test

Upload pdf files on those pages 
- Stock variations (Form A, Destruction Report, Incident Report)
- Supply chain  (VRF, PRE-ALERT)

Make sure you can make preview all of the pdf

## Print screen / video
<img width="1058" alt="Screenshot 2024-11-08 at 14 09 31" src="https://github.com/user-attachments/assets/10fada59-cd9b-46b6-942b-3b4a324f6ad2">


<img width="1091" alt="Screenshot 2024-11-08 at 14 09 42" src="https://github.com/user-attachments/assets/068a3deb-eac6-481b-b140-bd2b55dc7b0a">
